device/iot-device-client/src/main/java/com/microsoft/azure/sdk/iot/device/MessageProperty.java

```diff
@@ -163,17 +144,9 @@ public boolean hasSameName(String name) {
      *
      * @return whether the property is a valid application property.
      */
-    public static boolean isValidAppProperty(String name, String value) {
-        boolean propertyIsValid = false;
-
-        // Codes_SRS_MESSAGEPROPERTY_11_007: [The function shall return true if and only if the name and value only use characters in US-ASCII and the name is not a reserved property name.]
-        if (!RESERVED_PROPERTY_NAMES.contains(name)
-                && usesValidChars(name)
-                && usesValidChars(value)) {
-            propertyIsValid = true;
-        }
-
-        return propertyIsValid;
+    public static boolean isValidAppProperty(String name, String value)
+    {
+        return !RESERVED_PROPERTY_NAMES.contains(name);
```

Comment:
Do we need to validate even this?

Reply:
Maybe not, but that kind of change is a bit out of scope for the customer issue I'm working to solve. Maybe if a customer complains later, we can re-evaluate